dev/user.clj

```diff
@@ -36,3 +36,10 @@
   (println (ansi/yellow "Enter (help) for help"))
 
   :ready)
+
+
+(defn initialize-everything [username password name email]
```

Comment:
I'm somewhat hesitant about including this convenience function. The idea is to teach the user what each step is doing, via docs if necessary. Simple > Easy.

Reply:
Agree, that's just me debugging things and making it quicker for myself. Shouln't have been included